gradle/scripts/dependencyDefinitions.gradle

```diff
@@ -180,13 +180,16 @@ ext.externalDependency = [
     'reactivex': 'io.reactivex.rxjava2:rxjava:2.1.0',
     "slf4j": [
         "org.slf4j:slf4j-api:" + slf4jVersion,
-        "org.slf4j:slf4j-log4j12:" + slf4jVersion,
+        "org.slf4j:log4j-over-slf4j:" + slf4jVersion,
         "org.slf4j:jcl-over-slf4j:" + slf4jVersion
     ],
     "log4j": [
```

Comment:
is this block still needed if you removed references ?

Reply:
We still use log4j in unit test, and I keep the dependency for log4j in gobblin-aws since we specifically use appender which slf4j does not support.

Reply:
Gotcha. You may also want to check the conflicts, after which I can merge this so it could be tested in larger scope.